.circleci/config.yml

```diff
@@ -10,17 +10,20 @@ jobs:
           command: |
             TAG_VERSION=$(git tag -l --sort=-v:refname | head -n 1 | cut -d'-' -f2)
             PKG_VERSION=$(cat logdna/_version.py | grep "__version__ = " | cut -d'=' -f2 | cut -d"'" -f2)
-            if [[ "${TAG_VERSION}" != "${PKG_VERSION}" ]]; then
+            PKG_VERSION_TOML=$(cat pyproject.toml | grep "version = " | cut -d'=' -f2 | cut -d"'" -f2)
+
+            if [[ "${TAG_VERSION}" != "${PKG_VERSION}" ] -o ["${TAG_VERSION}" != "${PKG_VERSION_TOML}"]]; then
               echo "There is mismatch:"
               echo "  TAG_VERSION: ${TAG_VERSION}"
               echo "  PKG_VERSION: ${PKG_VERSION}"
+              echo "  PKG_VERSION_TOML: ${PKG_VERSION_TOML}"
               exit 1
             fi
       - run:
           name: Publish
           command: |
-            sudo pip install twine
-            python setup.py sdist
+            curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+            poetry build
```

Comment:
There is a command  `poetry publish --build`. I tested this command with test.pipy and it did not upload anything. So I am using poetry for creating the dist files and twine for uploading.